README.md

```diff
@@ -1,3 +1,19 @@
-# 3chLine
+# 3chPiazza
 
-a [Sails](http://sailsjs.org) application
+* tw*tter clone for 3ch
```

Comment:
"Communication "Piazza" for 3ch."  is just describing this product's feature, don't you?
i wanna remove other product's name :)